frontend_tests/node_tests/input_pill.js

```diff
@@ -60,12 +59,10 @@ run_test('basics', () => {
     blueslip.expect('error', 'Pill needs create_item_from_text');
     config.container = container;
     input_pill.create(config);
-    blueslip.reset();
 
     blueslip.expect('error', 'Pill needs get_text_from_item');
     config.create_item_from_text = noop;
     input_pill.create(config);
-    blueslip.reset();
```

Comment:
Hmm, I'm not sure I like removing the `blueslip.reset` calls in this sort of place -- we might want to name it differently, but structurally, we're asserting that the block of code between the expect and this line threw the target exception, and this API change seems to obscure that?  (Especially for the last `blueslip.except` in a test)